### DISCUSSION TITLE
Currently, I haven't found the relevant ARPACK GPU version online. When will you develop this version? Can you tell me if you have solved the GPU acceleration problem？

### DISCUSSION THREAD

**Discussion by xuanhaocheng** [opencollab/arpack-ng#464] [Announcements]
Hello,all contributors. I am an engineer, and the GPU accelerated solution of multiple eigenvalues and eigenvectors of a large-scale sparse matrix is of great significance for engineering. But currently, I haven't found the relevant ARPACK GPU version online. When will you develop this version?Or can you tell me if you have solved the GPU acceleration problem？

**Comment by fghoussen** [comment]
> I haven't found the relevant ARPACK GPU version

There is no such version.

> When will you develop this version?

This codebase is maintained only. No more development planned.

**Reply by sylvestre** [reply]
And I would probably start from scratch instead of trying to integrate it into arpack

**Comment by pasuka** [comment]
IMHO, it's better to discuss this proposal with [Spectra](https://github.com/yixuan/spectra) community. It's a modern rewrite of ARPACK and based on Eigen3 which supports CUDA.
